specs/lib/isDataUriSpec.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+describe('isDataUri', function() {
+    var isDataUri = require('../../lib/isDataUri');

```

Comment:
I tightened up the scope here.  Minor, but a good practice.  a3d64b64d499f6a65f49d8983f8d7885fcf99fd8